src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -145,6 +136,89 @@ public void RunApplication(Application app)
             }
         }
 
+        private async void ImportSettings(string fileContent)
+        {
+            var fileName = string.Empty;
+
+            using (FileStream fs = File.Create("preferenceSettings.xml"))
+            {
+                Byte[] title = new UTF8Encoding(true).GetBytes(fileContent);
+                fs.Write(title, 0, title.Length);
+                fileName = fs.Name;
+            }
```

Comment:
So we create a temp file and ask the js file to read the temp file for the settings?

Reply:
Would you explain to me again how this works? Is it possible that we still does not ask js to do the file parsing (for the security reason you mentioned) but call the API that Dynamo does not create a temp file?

Reply:
I'm creating a temp file because the Dynamo API requires the path string of the settings file. We can make another API method that skips the step that reads the file and goes directly to the content of the file.

Reply:
@filipeotero Thanks, I would love to have that instead of creating a temp file to eliminate the possibilities of permission issue